packages/react/src/components/Authenticator/ConfirmSignUp/ConfirmSignUp.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Can we turn off the rule that's requiring this?

Reply:
Yes but would like to handle it in a follow up PR since it will be applied to the rest of the _components_ directories that have already been updated to pass linting if that's cool?